Task1: print names, surnames and calculation

AndriiKolii opened the pull request:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

AndriiKolii committed:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

AndriiKolii committed:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

AndriiKolii committed:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

AndriiKolii committed:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

AndriiKolii committed:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

AndriiKolii committed:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

AndriiKolii committed:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

AndriiKolii committed:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

vloooo commented on kolii task1.py:
```diff
@@ -0,0 +1,7 @@
+print('Multi-paradigm languages: Task 1')
+print('Andrii Kolii FI-9119')
+print('Vladyslav Ovsianikov, ' * 44, 'Vladyslav Ovsianikov')
+result = ((15 + 11.1/3) / (33 + 12.3)) - 123.1
```
lost some spaces

vloooo commented on kolii task1.py:
```diff
@@ -0,0 +1,7 @@
+print('Multi-paradigm languages: Task 1')
+print('Andrii Kolii FI-9119')
+print('Vladyslav Ovsianikov, ' * 44, 'Vladyslav Ovsianikov')
```
extract value "Vladyslav Ovsianikov". you use it several times

vloooo commented:
never call the file as separate words.
split the words with underscore.
**kolii_task1**

AndriiKolii committed:
print course name
print name, surname, record book number
print lecturer name and surname 45 times
print calculation result (variant 7)

vloooo commented on kolii_task1.py:
```diff
@@ -0,0 +1,8 @@
+line = 'Vladyslav Ovsianikov, ' * 45
+result = ((15 + 11.1/3) / (33 + 12.3)) - 123.1
```
have missed one space, please check

vloooo commented on kolii_task1.py:
```diff
@@ -0,0 +1,8 @@
+print('Multi-paradigm languages: Task 1')
+print('Andrii Kolii FI-9119')
+print(line[:-2])
+print(result)
```
please add the blank lines in your code.
you can use them to divide the code for logic blocks.
For instance, you can divide this code for block of preparations and block of prints.

AndriiKolii committed:
print course name; student name, surname, record book number
input values and calculate result
